src/FA2.hs

```diff
@@ -0,0 +1,85 @@
+module FA2 
+  ( tokenMetadataRegistry
+  , addOperator
+  ) where
+
+import Text.Show (Show (..))
+
+import Lorentz
+import Lorentz.Contracts.Spec.FA2Interface hiding (name, sender) 
+
+import Util.Markdown
+import Util.Named
+import qualified Lorentz.Contracts.Spec.FA2Interface as FA2
+
+type Operators = BigMap (Address, Address) ()
+
+type Ledger = BigMap (Address, TokenId) Natural
```

Comment:
Another option is to have 2 `BigMap Address Natural` since there are only two token_ids. I am not sure what's better.

Reply:
Same as above for ledgers

Reply:
Btw, I don't see any updates of ledger_storage from the above link. Is it updated to store the same `(Address, (Address, TokenId))` key?